apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def

```diff
@@ -15,9 +15,11 @@
 # limitations under the License.
 
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.LogbackPatternConverterActivation
+toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.LogbackSkywalkingContextPatternConverterActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.mdc.MDCConverterActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.async.AsyncAppenderBaseInstrumentation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.async.LoggingEventInstrumentation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.logstash.TcpSocketAppenderActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.logstash.TraceIdJsonProviderActivation
+toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.logstash.SkywalkingContextJsonProviderActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.log.GRPCLogAppenderActivation
```

Comment:
Losing `org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.mdc.PrintMDCSkywalkingContextInterceptor` ?

Reply:
> Losing `org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.mdc.PrintMDCSkywalkingContextInterceptor` ?

I think only `*Instrumentation` and `*Activation` classes are needed in `*.def` file

Reply:
emmm..